clients/vault/src/replace.rs

```diff
@@ -9,7 +9,8 @@ use runtime::{
 	VaultRegistryPallet,
 };
 use service::{spawn_cancelable, Error as ServiceError};
-use wallet::StellarWallet;
+#[allow(unused_imports)]
+use wallet::{StellarWallet, keys::get_source_secret_key_from_env};
```

Comment:
Nit: If this import is only used for tests, we can also move it to the `tests` module.